core/src/main/scala/org/http4s/ContextService.scala

```diff
@@ -0,0 +1,26 @@
+package org.http4s
+
+import cats.data.{Kleisli, OptionT}
+import cats.Applicative
+
+object ContextService {
```

Comment:
We're moving away from the name "service".  We should consider how this interacts with #2567.

Reply:
I renamed that to ContextRoutes